### PR TITLE
Fix browser extension toggle button styles

### PR DESCRIPTION
**Pull request by erzhtor**
Seems like after converting `Toggle.scss` to `Toggle.module.scss` it got broken.

| Before | After|
| --: | :-- |
| image | image |

### CODE REVIEW

File: client/browser/src/browser-extension/manifest.spec.json [hunk 1]
```diff
@@ -29,7 +29,12 @@
   },
   "optional_permissions": ["tabs", "http://*/*", "https://*/*"],
   "content_security_policy": "script-src 'self' blob:; object-src 'self'",
-  "web_accessible_resources": ["img/*", "css/style.bundle.css", "css/branded-style.bundle.css"],
+  "web_accessible_resources": [
+    "img/*",
+    "css/style.bundle.css",
+    "css/branded-style.bundle.css",
+    "css/options.bundle.scss"
```
Comment (valerybugakov) [inline client/browser/src/browser-extension/manifest.spec.json:36]: Is `.scss` extension intentional here?

**Review by valerybugakov — APPROVED** [review]
Hey @erzhtor, thanks for looking into this! 

I want to understand the issue better to avoid having it with future refactorings because the CSS modules migration is not completed. How to prevent it from happening again?

**Review by tjkandala — APPROVED** [review]

**Comment by erzhtor** [comment]
> Hey @erzhtor, thanks for looking into this!
> 
> I want to understand the issue better to avoid having it with future refactorings because the CSS modules migration is not completed. How to prevent it from happening again?

Thanks for the review, @valerybugakov!

If I correctly understood, during codemod refactoring:
1. `Toggle.scss` was renamed to `Toggle.module.scss`and also was removed from `client/browser/src/branded.scss`, which is a manual listing of all styles to include in browser extension.
2. Previously, the main Bext webpack entry `options` popup didn't have any style output, because all React components which it included didn't have any imports of CSS, and we used to manually create a dedicated style entry `branded-style`.
3. However, with CSS modules files are added to corresponding react components, which now creates a style output for `options`, which we included in this PR.

In short, answering your question, it should not happen again (at least for Browser extension options popup) because all new refactorings will end up being automatically included in `options.bundle.css`

File: client/browser/src/browser-extension/manifest.spec.json (hunk 1 above)
Comment (erzhtor) [reply client/browser/src/browser-extension/manifest.spec.json]: Thanks, it is a typo, fixed.
